bin/postbuild.js

```diff
@@ -0,0 +1,93 @@
+const { last } = require("ramda")
```

Comment:
The existence of this file is a sign that we're out on deep water doing things much too custom. Nothing wrong with the script though (great that it's in js), more a comment on the difficulties of managing mono repos.

Reply:
I totally agree with you!

The community isn't really there yet... Mono-repos is kinda new and then add TypeScript on top of that. There's quite a few open issues on this in the TypeScript repo. Hopefully we can remove this in half a year.